includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -210,12 +205,15 @@ public function get_client() {
 		// Offline access so we can access the refresh token even when the user is logged out.
 		$this->google_client->setAccessType( 'offline' );
 		$this->google_client->setPrompt( 'consent' );
-
 		$this->google_client->setRedirectUri( $this->get_redirect_uri() );
-
 		$this->google_client->setScopes( $this->get_required_scopes() );
 		$this->google_client->prepareScopes();
 
+		$profile = ( new Profile( $this->user_options ) )->get();
```

Comment:
We should be able to use `$this->profile` here.

Reply:
```suggestion
		$profile = $this->profile->get();
```

Reply:
Good catch, updated!